src/sections/Learn-Layer5/Course-Overview/courseoverview.style.js

```diff
@@ -107,12 +107,40 @@ export const CourseOverviewWrapper = styled.div`
 
   .service-mesh-courses {
     display: inline-flex;
-    padding: 0 1rem;
+    padding: 0.4rem 0.75rem;
+    cursor: pointer;
+    border: 1px solid rgba(0,0,0,0.25);
+    border-radius: 5px;
     img {
       width: 2.5rem;
       height: 2.5rem;
     }
   }
+
+  .service-mesh-courses:first-child{
+    margin-right: 1rem;
+  }
+  .service-mesh-courses:hover{
+    box-shadow: rgb(0 0 0 / 8%) 0px 3px 10px 0px;
+  }
+  .resume-button-container{
+    display: inline-block;
+    margin-left: 1rem;
+    animation: bounce .5s 2;
```

Comment:
Instead of animating it, change the background color to `${props => props.theme.highlightColor}` which could also be done by just adding `secondary` prop to the `Button` component like:
`<Button secondary utl="...." title="....." />`

Reply:
Got it.